pkg/cluster/context.go

```diff
@@ -178,6 +178,15 @@ func (c *Context) Delete() error {
 	if err != nil {
 		return fmt.Errorf("error listing nodes: %v", err)
 	}
+
+	// clean up the kind kube config file that was created earlier
+	// also let the user know that they need to unset $KUBECONFIG to
+	// go back to their normal contexts
+	os.Remove(c.KubeConfigPath())
+	if os.Getenv("KUBECONFIG") == c.KubeConfigPath() {
+		fmt.Printf("kind kubectl config file removed, but you will need to unset KUBECONFIG to return to your normal contexts:\n\nunset KUBECONFIG\n")
```

Comment:
actually it should gracefully drop back to the default location if kubeconfig contains no files.

we also should really suggest the correct thing for windows as well, maybe just leave a todo here 🤔

Reply:
for win32:
`set KUBECONFIG=`

instead of `runtime.GOOS` abstraction we might as well just instruct:
>set the KUBECONFIG with value X`

>unset the KUBECONFIG`

Reply:
I'm not even sure we should mention this, does eg minikube? the workings of KUBECONFIG seem to be something orthogonal to learn vs kind 🤔

Reply:
minikube only has it in the docs, while kubeadm prints the stdout linux mini-guide on how to setup kubeconfig.

Reply:
maybe let's improve to a miniguide in a follow-up and handle windows too for set and unset